src/pretalx/schedule/models/schedule.py

```diff
@@ -48,11 +56,12 @@ class urls(EventUrls):
         public = "{self.event.urls.schedule}v/{self.url_version}/"
 
     @transaction.atomic
-    def freeze(self, name: str, user=None, notify_speakers: bool = True):
+    def freeze(self, name: str, comment: str, user=None, notify_speakers: bool = True):
```

Comment:
Please default to ``None`` here, as well, and put the comment to the end of the list of arguments. (I'm pretty sure we never call this method without explicit arguments in pretalx, but plugins may differ.)